js/src/FirstRun/TnC/tnc.js

```diff
@@ -34,6 +34,7 @@ export default function TnC ({ hasAccepted, onAccept }) {
     <div className={ styles.tnc }>
       <ReactMarkdown
         className={ styles.markdown }
+        escapeHtml={ false }
```

Comment:
Issue where react-markdown currently doesn't render properly. From their page -

"Inline HTML is currently broken for any tags that include attributes. A vague idea of how to fix this has been planned, but if you're feeling up to the task, create an issue and let us know!"